client/blocks/comment-detail/comment-detail-author.jsx

```diff
@@ -95,38 +152,50 @@ export class CommentDetailAuthor extends Component {
 					<div className="comment-detail__author-more-element">
 						<Gridicon icon="mail" />
 						<span>
-							{ authorEmail }
+							{ authorEmail || <em>{ translate( 'No email address' ) }</em> }
 						</span>
 					</div>
 					<div className="comment-detail__author-more-element">
 						<Gridicon icon="link" />
-						<ExternalLink href={ authorUrl }>
-							<Emojify>
-								{ urlToDomainAndPath( authorUrl ) }
-							</Emojify>
-						</ExternalLink>
+						{ !! authorUrl &&
+							<ExternalLink href={ authorUrl }>
+								<Emojify>
+									{ urlToDomainAndPath( authorUrl ) }
+								</Emojify>
+							</ExternalLink>
+						}
+						{ ! authorUrl &&
+							<em>{ translate( 'No website' ) }</em>
+						}
 					</div>
 					<div className="comment-detail__author-more-element">
 						<Gridicon icon="globe" />
 						<span>
-							{ authorIp }
+							{ authorIp || <em>{ translate( 'No IP address' ) }</em> }
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'IP address', 'NULL', { context: 'NULL'} )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/comments/user-info)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).